data/human/sales.txt

```diff
@@ -546,6 +546,9 @@ outfitter "FW Security"
 	"Tactical Scanner"
 	"Fragmentation Grenades"
 
+outfitter "Pirate Outfits" # Kept for backwards compatibility with plugins which used this before it was renamed to "Pirate Basics".
```

Comment:
```suggestion
outfitter "Pirate Outfits" # Kept for backwards compatibility with plugins that used it before it was renamed to "Pirate Basics."
```

You could maybe keep the period outside the quotes for clarity, but I think it's pretty clear even with them in, and that's better grammar.

Reply:
I believe ES tends to keep the full stop outside quotation marks? I don't remember anymore.

```suggestion
outfitter "Pirate Outfits" # Kept for backwards compatibility with plugins that used it before it was renamed to "Pirate Basics".
```

Reply:
Whoops, somehow I missed this comment, despite seeing the suggested change.

For future reference, ES follows American grammar rules, which means that periods and commas goes inside the quotation marks (but exclamation marks and question marks go inside or outside depending on other factors).

This is sort of an exception, since it's developer-facing rather than user-facing and is more clear than what would technically be proper grammar, so it's fine outside.